source/SH3/arc/vfile.cpp

```diff
@@ -100,3 +102,23 @@ std::size_t sh3_arc_vfile::ReadData(void* destination, std::size_t len, read_err
 
     return nbytes;
 }
+
+void sh3_arc_vfile::Dump2Disk()
+{
+    if(!open || buffer.empty())
```

Comment:
Not so much an error, it just doesn't dump anything to the disk to save a few clock cycles. I didn't really see the point of creating an empty file, though I did initially have it in.